### ISSUE TITLE
excluding "valid-lang" rule from a11ycheck

### ISSUE THREAD

**Issue by devarshipant** [dequelabs/axe-core#219]
Hi All,

On executing the script to exclude the valid-lang check, the lang test still runs in the console despite disabling it under 'rules.'

Any advise will be appreciated. 

thanks!

Here it is:
axe.a11yCheck(document, {
  runOnly: {
    type: "tag",
    values: ["wcag2a"]
  },
  "rules": {
    "color-contrast": { enabled: true },
    "valid-lang": { enabled: false },
  }
}, function(results) {
  console.log(results);
});

**Comment by marcysutton** [comment]
That rule was split in two and changed names. You found something we need to fix in our [documentation](https://github.com/dequelabs/axe-core/blob/master/doc/rule-descriptions.md)! Here are the correct rules to disable HTML lang:

`html-has-lang`
`html-lang-valid`

**Comment by devarshipant** [comment]
html-lang-valid or html-lang-invalid?

Thanks!

**Comment by marcysutton** [comment]
Actually, I misspoke. It was broken into three separate rules. `html-has-lang` and `html-lang-valid` check the HTML element, while `valid-lang` checks everything else. Can you elaborate with more details about the results returned?

**Comment by devarshipant** [comment]
I am interested in running snippet #4 under https://github.com/dequelabs/axe-core/blob/master/doc/API.md#options-parameter-examples -

{
  runOnly: {
    type: "tag",
    values: ["wcag2a"]
  },
  "rules": {
    "color-contrast": { enabled: true },
    "valid-lang": { enabled: false }
  }
}

I haven’t been able to run the set. In addition, when I pass the selector for the entire doc and enable all rules, it only runs under IE 9.

**Comment by marcysutton** [comment]
I ran a test against some basic HTML markup with that config and I don't see `valid-lang` running at all. Can you post the HTML you're running it against?  Here is my config, which I embedded in a script tag in an HTML document:

```javascript
var config = {
  runOnly: {
    type: "tag",
    values: ["wcag2a"]
  },
  "rules": {
    "color-contrast": { enabled: true },
    "valid-lang": { enabled: false }
  }
}
axe.a11yCheck(document, config, function(results) {
  console.dir(results.violations);
});
```

> In addition, when I pass the selector for the entire doc and enable all rules, it only runs under IE 9.

That sounds like a separate issue, definitely requiring more details.

**Comment by devarshipant** [comment]
Ok, the valid-lang works. I have attached the html file. I interesting to note the color contrast violation that gets flagged, which is same as given by WAT Analyzer. Also, note another rule I added “skip-link:” {enabled: true }, but did not see it flagged.

Thanks!
